qbft/spectest/tests/controller/processmsg/saved_broadcasted_decided.go

```diff
@@ -1,16 +1,19 @@
 package processmsg
 
 import (
+	"github.com/herumi/bls-eth-go-binary/bls"
+
 	"github.com/bloxapp/ssv-spec/qbft"
+	qbftcomparable "github.com/bloxapp/ssv-spec/qbft/spectest/comparable"
 	"github.com/bloxapp/ssv-spec/qbft/spectest/tests"
 	"github.com/bloxapp/ssv-spec/types"
 	"github.com/bloxapp/ssv-spec/types/testingutils"
-	"github.com/herumi/bls-eth-go-binary/bls"
 )
 
 // BroadcastedDecided tests broadcasting decided
 func BroadcastedDecided() tests.SpecTest {
```

Comment:
Except the input value, what is the difference between this and fully_decided?

Conceptually those are seperated cases... practically one single test covers both

Reply:
No need to take action for now